recipe/meta.yaml

```diff
@@ -106,6 +106,7 @@ requirements:
     # other requirements
     - python 3.12  # [megabuild]
     - python       # [not megabuild]
+    - wheel
```

Comment:
not necessary; it was already fixed by the pip PR I mentioned

Reply:
It's better to be explicit about the dependencies we use.